.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[ci skip]')"
     strategy:
       matrix:
-        php: ['7.3', '7.4']
+        php: ['7.3', '7.4', '8.0', '8.1', '8.2']
```

Comment:
Thanks for the PR @lukasbesch !

Could you remove the old 7.x versions?
Best we don't support that any more